### DISCUSSION TITLE
on_hover effect stays even after cursor has left the control.

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#1010] [Q&A]
I have created a Container and I have given it a hover effect which increases the size of the container and changes its color, the problem is that when I hover the size and color change as expected but when I leave the container the hover effect still stays, and it does not return to the default size and color, is there something I'm doing wrong? Should any additional info be needed I shall provide it ASAP.
Any help would be appreciated.

**Comment by ndonkoHenri** [comment]
Have a look at what I did here: https://github.com/ndonkoHenri/Flet-Samples/blob/1882ae23061f0aba801e624479510d9f6ce6f5ec/Random%20Image%20Viewer/utils.py#L97

If it still doesnt help, then build up a little sample code, so I can help.
